plugins/score-card/src/components/ScoreCardTable/ScoreCardTable.tsx

```diff
@@ -44,56 +45,57 @@ const useScoringAllDataLoader = () => {
 
 type ScoreTableProps = {
   title?: string;
-  scores: SystemScoreExtended[];
+  scores: EntityScoreExtended[];
 };
 
 export const ScoreTable = ({ title, scores }: ScoreTableProps) => {
-  const columns: TableColumn<SystemScoreExtended>[] = [
+  const columns: TableColumn<EntityScoreExtended>[] = [
     {
       title: 'Name',
-      field: 'systemEntityName',
-      render: systemScore =>
-        systemScore.catalogEntityName ? (
-          <Link
-            to={`/catalog/${systemScore.catalogEntityName.namespace}/${systemScore.catalogEntityName.kind}/${systemScore.catalogEntityName.name}/score`}
-            data-id={systemScore.systemEntityName}
+      field: 'entityName',
+      render: entityScore => {
+        if (!entityScore.entityRef?.name) {
+          return <>Missing entityRef.name key</>
+        }
+
+        return (<Link
+          to={`/catalog/${entityScore.entityRef.namespace ?? DEFAULT_NAMESPACE}/${entityScore.entityRef.kind}/${entityScore.entityRef.name}/score`}
+          data-id={entityScore.entityRef.name}
           >
-            {systemScore.systemEntityName}
-          </Link> // NOTE: we can't use EntityRefLink as it does not yet support navigating to "/score" (or other tab) yet
-        ) : (
-          <>{systemScore.systemEntityName}</>
-        ),
+            {entityScore.entityRef.name}
```

Comment:
Thinking of adding a new column with "kind" and/or adding the kind here like
```suggestion
            {entityScore.entityRef.kind}:{entityScore.entityRef.name}
```

Reply:
I prefer a new column so we can keep sorting by name.

Reply:
Thanks, perfect!